packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -132,94 +110,443 @@ void main() {
-        createImageAnalysis:
-            (ResolutionSelector? resolutionSelector, int? targetRotation) =>
-                ImageAnalysis.detached(
-                    resolutionSelector: resolutionSelector,
-                    initialTargetRotation: targetRotation),
-        createResolutionStrategy: (
-            {bool highestAvailable = false,
-            Size? boundSize,
-            int? fallbackRule}) {
-          if (highestAvailable) {
-            return ResolutionStrategy.detachedHighestAvailableStrategy();
-          }
-          return ResolutionStrategy.detached(
-              boundSize: boundSize, fallbackRule: fallbackRule);
-        },
-        createResolutionSelector: (ResolutionStrategy resolutionStrategy,
-                ResolutionFilter? resolutionFilter,
-                AspectRatioStrategy? aspectRatioStrategy) =>
-            ResolutionSelector.detached(
-                resolutionStrategy: resolutionStrategy,
-                resolutionFilter: resolutionFilter,
-                aspectRatioStrategy: aspectRatioStrategy),
-        createFallbackStrategy: (
-                {required VideoQuality quality,
-                required VideoResolutionFallbackRule fallbackRule}) =>
-            FallbackStrategy.detached(
-                quality: quality, fallbackRule: fallbackRule),
-        createQualitySelector: (
-                {required VideoQuality videoQuality,
-                required FallbackStrategy fallbackStrategy}) =>
-            QualitySelector.detached(qualityList: <VideoQualityData>[
-          VideoQualityData(quality: videoQuality)
-        ], fallbackStrategy: fallbackStrategy),
-        createCameraStateObserver: (_) => MockObserver(),
-        requestCameraPermissions: (_) => Future<void>.value(),
-        startListeningForDeviceOrientationChange: (_, __) {},
-        setPreviewSurfaceProvider: (_) => Future<int>.value(
-            3), // 3 is a random Flutter SurfaceTexture ID for testing,
-        createAspectRatioStrategy: (int aspectRatio, int fallbackRule) =>
-            AspectRatioStrategy.detached(
-                preferredAspectRatio: aspectRatio, fallbackRule: fallbackRule),
-        createResolutionFilterWithOnePreferredSize:
-            (Size preferredResolution) =>
-                ResolutionFilter.onePreferredSizeDetached(
-                    preferredResolution: preferredResolution),
-        getCamera2CameraInfo: (_) =>
-            Future<Camera2CameraInfo>.value(MockCamera2CameraInfo()),
-        getUiOrientation: () =>
-            Future<DeviceOrientation>.value(DeviceOrientation.portraitUp),
-        previewSurfaceProducerHandlesCropAndRotation: (_) =>
-            Future<bool>.value(false),
-      );
+    MockProcessCameraProvider mockProcessCameraProvider, {
+    ResolutionFilter Function({
+      required CameraSize preferredSize,
+      // ignore: non_constant_identifier_names
+      BinaryMessenger? pigeon_binaryMessenger,
```

Comment:
Can we just change the name versus the ignores?

Reply:
Unfortunately, no because this is a named parameter. You an only rename positional parameters. It is on my radar to either change these names to follow the languages style.